README.md

```diff
@@ -39,13 +39,33 @@ All messages                         | Alpha    |
 (See [maturity-matrix.yaml](https://github.com/open-telemetry/community/blob/47813530864b9fe5a5146f466a58bd2bb94edc72/maturity-matrix.yaml#L57)
 for definition of maturity levels).
 
-Note that maturity guarantees apply only to wire-level compatibility for the binary
-Protobuf serialization. Neither message, field, nor enum names of Protobuf messages
-are visible on the wire and are not considered part of the guarantees. We are free
-to make a change to the names.
+## Stability Definition
 
-In the future when OTLP/JSON is declared stable, field names will also become part of
-the maturity guarantees, since field names are visible on the wire for JSON encoding.
+Components marked `Stable` provide the following guarantees:
+
+- Field types will not change.
+- Field numbers will not change.
+- Numbers assigned to enum choices will not change.
+- Service names and service package name will not change.
+- Service operation names, parameters names and types will not change.
```

Comment:
Why params name (actually what is the param name)?

```proto
service LogsService {
  // For performance reasons, it is recommended to keep this RPC
  // alive for the entire life of the application.
  rpc Export(ExportLogsServiceRequest) returns (ExportLogsServiceResponse) {}
}
```

Reply:
Hmm, not names. It should be parameter and return types. Fixed.